raiden/tests/mocked/test_matrix_transport.py

```diff
@@ -434,24 +577,21 @@ def test_retry_queue_does_not_resend_removed_messages(
     # greenlet to run but instead manually call its `_check_and_send()` method.
     retry_queue = _RetryQueue(transport=mock_matrix, receiver=Address(factories.HOP1))
 
-    message = make_message()
+    message_event = make_message_event(recipient=Address(factories.HOP1))
+    message = make_message(message_event=message_event)
     serialized_message = MessageSerializer.serialize(message)
-    queue_identifier = QueueIdentifier(
-        recipient=Address(factories.HOP1),
-        canonical_identifier=CANONICAL_IDENTIFIER_UNORDERED_QUEUE,
-    )
-    retry_queue.enqueue(queue_identifier, [(message, None)])
 
-    # TODO: Fix the code below, the types are not matching.
-    mock_matrix._queueids_to_queues[queue_identifier] = [message]  # type: ignore
+    retry_queue.enqueue(message_event.queue_identifier, [(message, None)])
```

Comment:
Yes, since the metadata is only relevant for batching (not utilised here because only 1 message) and inside of the `_send_raw`.
The acceptance criteria for this test is just the `_send_raw` being called with the message, so it's fine.
But I see that this can become a problem once the mocking is changed without looking at this test.